Doc/library/turtle.rst

```diff
@@ -1822,7 +1831,10 @@ Window control
    Set background image or return name of current backgroundimage.  If *picname*
    is a filename, set the corresponding image as background.  If *picname* is
    ``"nopic"``, delete background image, if present.  If *picname* is ``None``,
-   return the filename of the current backgroundimage. ::
+   return the filename of the current backgroundimage.
+
+   .. doctest::
+      :skipif: True
```

Comment:
Maybe we should just not add the `.. doctest` directive here, if we never want sphinx to actually run this REPL snippet as a doctest?

Reply:
If I remove this line, there will be an error locally. I don't know why, I have no idea how to debug this kind of problems.